rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,165 @@
+
+# Detailed Design
+[design]: #detailed-design
+
+## Goals
+
+- Adopt the *Memorandum of Understanding on Equitable Moderation* to foster a shared understanding of equitable moderation practices that accommodate a community with diverse sociopolitical views.
+- Ensure transparent accountability of moderation practices to community members.
+- Ground community member accountability in publicly available general principles, providing a stable basis for behavioral expectations.
+- Make wide-ranging decisions regarding community member exclusion understandable.
+- Promote a diverse moderation team to balance diverse perspectives on controversial issues.
+- Establish an *Appeals Council* to hold moderation team members and procedures accountable and to allow for the revision of significant decisions if they fail to meet the community's reasonable standards.
+
+## Memorandum of Understanding on Equitable Moderation
+
+In response to the impression left by past events and with a fervent desire for improvement, we adopt the following principles and maxims as the *Memorandum of Understanding on Equitable Moderation*. These principles serve to clarify the overlapping consensus on reasonable expectations for community participation:
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
+5) We affirm that moderation actions must be fair and proportionate to the related issue.
+6) We ensure that concrete reasons for permanent bans are available for inspection by long-standing community members.
+7) We recognize that moderation actions should not be based solely on the volume of complaints.
+8) We commit to treating community members with respect and avoiding derogatory language when addressing concerns.
+9) We affirm our role as moderators is to facilitate a respectful community, not to punish members. We will use language that reflects mutual respect.
+10) We prefer inclusion to exclusion, resorting to permanent exclusion only in cases of extreme necessity. We are committed to exploring ways to prevent exclusion and to engaging constructively in efforts to reinstate excluded members.
```

Comment:
This also needs to respect the implicit exclusion of community members that then seemingly-voluntarily retreat from community spaces, just because the presence of certain members that is hugely deterring to a subset of folks needs to be endured just for the sake of reinstating them.